bundle-workflow/src/run_integ_test.py

```diff
@@ -86,6 +86,7 @@ def main():
             args.s3_bucket, args.build_id, args.opensearch_version, args.architecture, work_dir)
         pull_common_dependencies(work_dir, build_manifest)
         sync_dependencies_to_maven_local(work_dir, build_manifest.build.version)
+        failed_components = dict()
```

Comment:
I would collect all results, then log both, not just failures. It's equally important to know what passed.

Reply:
The error logs are already getting print here: https://github.com/opensearch-project/opensearch-build/blob/main/bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py#L117.

`failed_components ` is just to avoid showing status code as 0 for any plugin failure

Reply:
But I like the summary that I can easily scroll to at the end!